libfwupdplugin/fu-context.c

```diff
@@ -32,6 +32,7 @@ typedef struct {
 	FuLidState lid_state;
 	guint battery_level;
 	guint battery_threshold;
+	gboolean me_locked;
```

Comment:
ME is only specific to Intel - this shouldn't be in the context for all of FuContextPrivate.